components/tokens/custom-spectrum/custom-vars.css

```diff
@@ -25,4 +25,8 @@ governing permissions and limitations under the License.
   --spectrum-line-height-large: 1.7;
   --spectrum-line-height-medium: 1.5;
   --spectrum-line-height-small: 1.3;
+
+  /* missing token for button control size small - used in custom-medium-vars and custom-large-vars */
+  --spectrum-radio-button-control-size-small-desktop: 12px;
+  --spectrum-radio-button-control-size-small-mobile: 16px;
```

Comment:
Are these not coming through in the tokens package? It looks like they're defined here: https://github.com/adobe/spectrum-tokens/blob/67df19bf39a71f6dbd0df9b64642e057fb1c9295/src/component-layout.json#L1043-L1054

`--spectrum-radio-button-control-size-small`

Reply:
I see, there was some confusion for me since the medium/large/extra large sizes needed additional tokens, but it does appear that small is already available. Thanks! New commit should resolve this.